README.md

```diff
@@ -350,6 +350,22 @@ Check out the samples project here, with instructions on how to run them on Kube
 - [JavaScript sample](https://github.com/cloudstateio/samples-js-chat)
 - [Java sample](https://github.com/cloudstateio/samples-java-chat)
 
+### Building and publishing a sample application
+
+The sample application is packaged in a Docker container and pushed to the Docker repository path formed using the system properties `docker.registry` and `docker.username`. The default repository path is `cloudstateio/<application-name>:tag`
```

Comment:
I did not know that one can use system properties here. I thought that the registry and such settings are "hard coded" in build.sbt.

Reply:
You can, I set them to publish the app to my own dockerhub acccount instead of the Cloudstate.io one.